ejson/src/ejson-payload-converter.ts

```diff
@@ -33,7 +34,8 @@ export class EjsonPayloadConverter implements PayloadConverterWithEncoding {
 
     return {
       metadata: {
-        [METADATA_ENCODING_KEY]: u8('ejson/plain'),
+        [METADATA_ENCODING_KEY]: u8('json/plain'),
+        format: u8('extended'),
```

Comment:
In case user wants to create their own Composite converter that can differentiate between EJSON and JSON

Reply:
Let’s explain this in code and add an explanation in the readme of how to migrate from default converter